doc/ug/bazel_notes.md

```diff
@@ -272,6 +272,17 @@ test --test_tag_filters=-cw310
 See the [`.bazelrc`](https://github.com/lowRISC/opentitan/blob/master/.bazelrc) file in the OpenTitan repository for more examples.
 Additionally, for more information see the Bazel [documentation](https://bazel.build/run/bazelrc).
 
+### Site-specific `.bazelrc-site` options
+
+In addition to each user's `$HOME/.bazelrc` file and the project-level configurations in [`$REPO_TOP/.bazelrc`](https://github.com/lowRISC/opentitan/blob/master/.bazelrc), there is the option to add site-specific configuration options, by adding them to the file `$REPO_TOP/.bazelrc-site`.
+
+For groups working on shared, site-specific servers, the `bazelrc-site` file can be useful for enforcing policies such as:
+- Default locations for build artifacts (e.g., new defaults for `--user-output-base` options, outside the "$HOME" filesystem).
+- Site-specific, non-standard paths for build-tools, libraries, or package configuration data.
+
+The policies and paths for each contributing site could vary greatly.
+So in order to avoid accidental conflicts between sites, the `.bazelrc-site` file is explicitly _not_ included in the git repository.
```

Comment:
Nit: I might point out that this is enforced by CI with `ci/scripts/check-no-bazelrc-site.sh`.

Reply:
Done!